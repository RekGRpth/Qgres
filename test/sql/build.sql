\set ECHO none

\i test/pgxntool/psql.sql

BEGIN;
CREATE EXTENSION IF NOT EXISTS citext;

SET client_min_messages = WARNING;
\i sql/qgres.sql

\echo # TRANSACTION INTENTIONALLY LEFT OPEN!

-- vi: expandtab ts=2 sw=2
