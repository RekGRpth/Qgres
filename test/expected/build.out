\set ECHO none
 role__create 
--------------
 
(1 row)

 role__create 
--------------
 
(1 row)

 role__create 
--------------
 
(1 row)

# TRANSACTION INTENTIONALLY LEFT OPEN!
